CRM/Civixero/Invoice.php

```diff
@@ -66,7 +66,7 @@ public function pull(array $params): int {
           'accounts_needs_update' => FALSE,
         ];
 
-        $prefix = $this->getSetting('xero_invoice_number_prefix') ?: '';
+        $prefix = \Civi::settings()->get('xero_invoice_number_prefix');
```

Comment:
This ignores the function intended to get connection specific settings

Reply:
Ah ok, that setting is not actually supplied by the connector though? Happy to change it back though - I probably thought it was simpler to avoid another level of abstraction but maybe makes sense here